src/js/bootstrap.js

```diff
@@ -139,9 +139,7 @@
     }
 
     videojs.log.debug = makeCachedLogger();
-    videojs.log.warn = makeCachedLogger();
```

Comment:
we're losing the ability to cache error and warn. We should make `makeCachedLogger` also call the original function as well.